src/server/src/api/__tests__/insert.test.ts

```diff
@@ -65,27 +66,103 @@ describe('insert build handler', () => {
     });
   });
 
-  describe('onInserted', () => {
-    test('called with a comparator of the inserted branch against its parentRevision', () => {
-      const handleInsert = jest.fn(() => Promise.resolve());
-      const handler = insertBuild(queries, config, handleInsert);
-      app.post('/test', handler);
+  describe('parent revision', () => {
```

Comment:
The changes inside this function are white-space only

Reply:
In the gear icon next to "Jump to..."
![image](https://user-images.githubusercontent.com/1002461/74512474-a2ed1300-4f08-11ea-9401-8f614646c4bb.png)

Reply:
Oh thanks!